TestCases/compliance-level-3/0084-feel-for-loops/0084-feel-for-loops.dmn

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0084-feel-for-loops" name="0084-feel-for-loops" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <description>FEEL for loops</description>
+
+    <decision name="decision_001" id="_decision_001">
+        <variable name="decision_001"/>
+        <literalExpression>
+            <text>for i in [1,2,3] return i + 1</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_002" id="_decision_002">
+        <variable name="decision_002"/>
+        <literalExpression>
+            <text>for i in [1,2,3], j in [4,5] return i + j</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_003" id="_decision_003">
+        <variable name="decision_003"/>
+        <literalExpression>
+            <text>for i in [2..4] return i</text>
```

Comment:
In the DMNv1.2 I read:
![image](https://user-images.githubusercontent.com/1699252/51469419-9f24a680-1d71-11e9-8c3d-a9b790df6d87.png)

hence this is not a valid test, it should be rewritten as:
```
for i in 2..4 return i
```

at least this is my interpretation of the spec v1.2

Reply:
Thank @tarilabs .  I do recall this.  I was pretty sure I put in tests for both syntaxes actually - kind of expecting the [] one to be a discussion point.  I sure meant to anyways.  I recall reading in the spec that it was without [] and thought that odd since ranges elsewhere exist inside [].  I agree that is what the spec says.  I'll amend.  Thanks.

Reply:
oh, I *did* put in both - `decision_007` and `decision_008` make the assertions without [].  I think it probably best to support both (I do) but, will remove those tests.  Thanks again.

Reply:
Actually.  I remember now exactly why I put these in.  Despite what the spec says in that particular place, there is the 'syntactic sugar' section that gives addition usages on 'in'.  The [x..y] notation is one of those.

Before we lose those tests, perhaps we ought to open this question out to @agilepro and @etirelli and anyone else paying attention.  My interpretation is that it is likely valid syntax.

Reply:
@StrayAlien the 'for' loop syntax does not support ranges with [ ] . I think the reason is because in the case of 'for' it wouldn't make sense to use exclusive ranges, only inclusive. As usual, we can open a ticket to the RTF to add support for the syntax, but it is not in DMN 1.2.